src/Serilog.Sinks.Splunk/Sinks/Splunk/EventCollectorRequest.cs

```diff
@@ -32,6 +32,11 @@ internal SplunkEvent(string logEvent, string source, string sourceType, string h
                 jsonPayLoad = jsonPayLoad + @",""index"":""" + index + @"""";
```

Comment:
Another thing to consider is adding support for batching i.e. sending multiple events in a single request for efficiency. That would require having some sort of Flush() method. See [here](http://dev.splunk.com/view/event-collector/SP-CAAAE6P) under Examples for how batching works.

Our logging [library](https://github.com/splunk/splunk-library-dotnetlogging/blob/master/src/Splunk.Logging.Common/HttpEventCollectorSender.cs) handles a lot of this natively (including retries, batching, async), but it does't support DotNet core.